conda/recipes/cudf_kafka/meta.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set cuda_version='.'.join(environ.get('CUDA_VERSION', '10.0').split('.')[:2]) %}
+
+package:
+  name: cudf_kafka
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - CC
+    - CXX
+    - CUDAHOSTCXX
+    - PARALLEL_LEVEL
+    - VERSION_SUFFIX
+
+requirements:
+  build:
+    - cmake >=3.14.0
+  host:
+    - libcudf_kafka {{ version }}
+    - cudf {{ version }}
+    - python-confluent-kafka
+    - cython >=0.29,<0.30
+    - setuptools
+    - numpy
```

Comment:
Is `numpy` used directly in `cudf_kafka`? Otherwise just rely on `cudf` to pull it.